build/azure-pipelines/common/publish.ts

```diff
@@ -1075,7 +1075,8 @@ async function main() {
 	console.log(`All ${done.size} artifacts published!`);
 }
 
-if (require.main === module) {
+const normalizeScriptPath = (p: string) => p.replace(/\.(js|ts)$/, '');
+if (normalizeScriptPath(import.meta.filename) === normalizeScriptPath(process.argv[1])) {
```

Comment:
In the future we should be able to replace this with `if (import.meta.main)` once it's been out a bit long. It landed in a pretty recent version of node 22